pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/queries/QueryRunnerTestBase.java

```diff
@@ -197,14 +199,17 @@ protected List<Object[]> queryH2(String sql)
     return result;
   }
 
-  protected void compareRowEquals(ResultTable resultTable, List<Object[]> expectedRows) {
-    compareRowEquals(resultTable, expectedRows, false);
+  protected void compareRowEquals(ResultTable resultTable, List<Object[]> expectedRows, String sql) {
```

Comment:
not sure this change is necessary. in mvn tests the query will be part of the test name.
if we want to be more verbose here suggest take a stab into how test suite utilizes the parameterized test input

Reply:
I've added that because in Intellij query is not added, which makes it difficult to find the query that fail. Also the description, if defined, should be part of the test name

Reply:
Sorry, I though you were complaining about the name of the tests. In this case... I definitively missed the query in the assertion.

Reply:
ah. i dont use intellij ui. it is printed in the test report from mvn test :-)

Reply:
but regardless, i think bottomline is can we agree to separate into a different PR (all the test improvements)